compiler/codegen/library.cpp

```diff
@@ -306,6 +344,46 @@ void codegen_library_python(std::vector<FnSymbol*> functions) {
   }
 }
 
+void codegen_library_fortran(std::vector<FnSymbol*> functions) {
+  if (fLibraryCompile && fLibraryFortran) {
+    setupFortranTypeMap();
+    makeFortranModule(functions);
+  }
+}
+
+void makeFortranModule(std::vector<FnSymbol*> functions) {
+  const char* filename = fortranModulename[0] != '\0' ? fortranModulename
+                                                      : libmodeHeadername;
+  int indent = 0;
+  fileinfo fort = { NULL, NULL, NULL };
+
+  openLibraryHelperFile(&fort, filename, "f90");
+
+  if (fort.fptr != NULL) {
+    FILE* save_cfile = gGenInfo->cfile;
+    gGenInfo->cfile = fort.fptr;
```

Comment:
Could you add a comment to codegen.h near the declaration of `cfile` that it should be renamed? E.g. `// TODO: Rename cfile to just file since it's used when generating Fortran or Python interfaces`.